praw/models/reddit/wikipage.py

```diff
@@ -68,6 +91,32 @@ def remove(self, redditor: "praw.models.Redditor"):
         )
         self.wikipage._reddit.post(url, data=data)
 
+    def revert(self):
+        """Revert a wikipage back to a specific revision.
+
+        To revert the page ``"praw_test"`` in ``r/test`` to revision ``[ID]``, try
+
+        .. code-block:: python
+
+            reddit.subreddit("test").wiki["praw_test"].revision("[ID]").mod.revert()
+
+        .. note::
+
+            When you attempt to revert the page ``config/stylesheet``, Reddit checks to
+            see if the revision being reverted to passes the CSS filter. If the check
+            fails, then the revision attempt will also fail. For example, you can't
+            revert to a revision that contains a link to ``url(%%PRAW%%)`` if there is
+            no image named ``PRAW`` on the current stylesheet.
```

Comment:
Could we explain how to catch the error that would be raised? From what you said and looking at the test case, this would raise a `Forbidden` error which doesn't accurately represent the reason why it actually failed.